src/Maui.Plugins.PageResolver/Resolver.cs

```diff
@@ -1,10 +1,27 @@
 ﻿using Microsoft.Extensions.DependencyInjection;
 using System;
+using Microsoft.Maui.Hosting;
 
 namespace Maui.Plugins.PageResolver
 {
     public static class Resolver
     {
+        internal class Initializer : IMauiInitializeService
```

Comment:
Personally not a fan of nested classes. Can this be moved outside of the Resolver class?

Reply:
I put it in there to give it access to the private scope field. But nothing forces it to stay there other than personal preference. I'll move the null check into the RegisterServiceProvider method.

Reply:
Done